### ISSUE TITLE
Accessibility issue: use of "another website" as link text

### ISSUE THREAD

**Issue by frankieroberto** [alphagov/government-frontend#3425]
Consultations and call for evidence pages sometimes link out to other website where the consultation is taking place.

In this instances, this text is shown, with "another website" being the link text:

![Screenshot 2024-11-18 at 16 05 35](https://github.com/user-attachments/assets/45b115c0-9b5b-407b-9bc8-d81703dc2e42)

This link text might not make sense out of the context of the paragraph, and possibly fails WCAG 2.4.4: [Link Purpose](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)?
